build_variables.bzl

```diff
@@ -554,6 +554,8 @@ torch_mobile_core = [
     # TODO: Remove this dependency
     "torch/csrc/jit/backends/backend_debug_info.cpp",
     "torch/csrc/jit/mobile/compatibility/model_compatibility.cpp",
+    # TODO: This line needs to be uncommented to build mobile in OSS with flatbuffers
```

Comment:
remove the commented code

Reply:
It's actually intentional..we need to create a pull request to land the change by uncommenting this line. If anyone is interested, it can be a self-contained change too